src/app/pages/rss/rss-feeds/rss-feeds.component.html

```diff
@@ -0,0 +1,17 @@
+<div class="rss-feed__card" *ngIf="rssData">
+  <div *ngFor="let rssItem of rssData.items; let i = index">
```

Comment:
your card starts here or in the .rss-feed__card-content div ?

Reply:
here!